data/elasticsearch/elasticsearch.go

```diff
@@ -23,26 +24,67 @@ func (es *ElasticSearch) Connect(url string) error {
 	return err
 }
 
-// Index stores a document inside elastic search
-func (es *ElasticSearch) Index(esIndex string, esType string, body interface{}) error {
+// CreateIndexIfNotExists Creates an index if it doesn't already exists
```

Comment:
You're right. This is just from previous experimentation, but this code is not called. Do you want me to remove it ?